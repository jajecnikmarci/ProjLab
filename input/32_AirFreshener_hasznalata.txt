starttest
add room room1 3
add room room2 3 poisonous
add door room1 room2 door1
add item room1 AirFreshener airFreshener
add item room1 FFP2 ffp2
add person room1 Student student1
pickup student1
pickup student1
gotoroom student1 room2
printstate room2
use student1 airFreshener
printstate room2
endtest