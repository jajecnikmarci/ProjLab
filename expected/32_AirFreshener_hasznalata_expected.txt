room2
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
	- poisonEffects: 
		poison1
	- stunEffects: 
	- lastCleaning: null
room2
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null

